source/index.js

```diff
@@ -236,14 +236,14 @@ module.exports = async (input = 'patch', options) => {
 	tasks.add({
 		title: 'Pushing tags',
 		skip: async () => {
-			if (!(await git.hasUpstream())) {
-				return 'Upstream branch not found; not pushing.';
-			}
-
 			if (options.preview) {
 				return '[Preview] Command not executed: git push --follow-tags.';
 			}
 
+			if (!(await git.hasUpstream())) {
```

Comment:
I'm not sure we want to preview the command in case it won't run successfully. Is there any specific reason for this change?